source/appModules/soffice.py

```diff
@@ -1,82 +1,23 @@
-#appModules/soffice.py
-#A part of NonVisual Desktop Access (NVDA)
-#This file is covered by the GNU General Public License.
-#See the file COPYING for more details.
-#Copyright (C) 2006-2019 NV Access Limited, Bill Dengler
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2006-2021 NV Access Limited, Bill Dengler, Leonard de Ruijter
 
 from comtypes import COMError
-from comInterfaces import IAccessible2Lib as IA2
-import IAccessibleHandler
+from IAccessibleHandler import IA2, splitIA2Attribs
 import appModuleHandler
 import controlTypes
 import textInfos
 import colors
 from compoundDocuments import CompoundDocument
-from NVDAObjects.JAB import JAB, JABTextInfo
 from NVDAObjects.IAccessible import IAccessible, IA2TextTextInfo
 from NVDAObjects.behaviors import EditableText
 from logHandler import log
+import speech
+import api
+import braille
+import vision
 
-def gridCoordStringToNumbers(coordString):

```

Comment:
Is the removal of these functions and classes going to affect add-ons?

Reply:
That's very unlikely I think. Add-ons should avoid using appModules directly.

Reply:
@feerrenrut @michaelDCurran  what do you think?

Reply:
@feerrenrut  - requested you review pending this discussion